packages/dd-trace/test/appsec/iast/analyzers/insecure-cookie-analyzer.spec.js

```diff
@@ -43,6 +43,12 @@ describe('insecure cookie analyzer', () => {
         res.setHeader('set-cookie', ['key=value; HttpOnly', 'key2=value2; Secure'])
       }, INSECURE_COOKIE, 1)
 
+      testThatRequestHasVulnerability((req, res) => {
+        const cookieNamePrefix = (new Array(50)).join('0')
```

Comment:
Maybe it would be clearer `'0'.repeat(50)`

Reply:
Same in [packages/dd-trace/test/appsec/iast/analyzers/no-samesite-cookie-analyzer.spec.js](https://github.com/DataDog/dd-trace-js/pull/4764/files#diff-fa8ab420938d6711dbcfd5529f14d60b609a80cdc367b5920962fb1a78234d00R70) and [packages/dd-trace/test/appsec/iast/analyzers/no-httponly-cookie-analyzer.spec.js](https://github.com/DataDog/dd-trace-js/pull/4764/files#diff-96883fba5aaa9ee6fc8d343d54f44d6e88d9d8face0dc2646ceca3a901670772R59)